openhtf/io/output/mfg_inspector.py

```diff
@@ -37,7 +36,7 @@
 import threading
 import zlib
 
-from openhtf.io.output import json_factory
```

Comment:
you can still leave this as `from openhtf.io.output import json_factory`, `from X import Y` syntax is allowed as long as Y is a module (ie, style forbids importing individual symbols from a within a module that way).